metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/hook/ingestion/IngestionSchedulerHook.java

```diff
@@ -75,10 +75,19 @@ public void invoke(@Nonnull MetadataChangeLog event) {
    * of an Ingestion Source Info aspect, which in turn contains the schedule associated with the source.
    */
   private boolean isEligibleForProcessing(final MetadataChangeLog event) {
+    return isIngestionSourceUpdate(event) || isIngestionSourceDelete(event);
+  }
+
+  private boolean isIngestionSourceUpdate(final MetadataChangeLog event) {
     return Constants.INGESTION_INFO_ASPECT_NAME.equals(event.getAspectName())
-        && (ChangeType.DELETE.equals(event.getChangeType())
-        || ChangeType.UPSERT.equals(event.getChangeType())
-        || ChangeType.CREATE.equals(event.getChangeType()));
+        && (ChangeType.UPSERT.equals(event.getChangeType())
+        || ChangeType.CREATE.equals(event.getChangeType())
+        || ChangeType.DELETE.equals(event.getChangeType()));
```

Comment:
did you mean to level `DELETE` here?

Reply:
level?

Reply:
leave***

Reply:
THis is actually here just for safety. If someone issues their own individual aspect delete then we will still appropriately unschedule. This should not happen during our own system's normal operation as the contract currently is implemented